pkg/utils/apiresources.go

```diff
@@ -44,41 +44,54 @@ var (
 	apiGroupSepToken = ";"
 )
 
-// DisabledResourceConfig represents the configuration that identifies the API resources should not be selected.
-type DisabledResourceConfig struct {
-	// groups holds a collection of API group, all resources under this group will be avoided.
+// ResourceConfig represents the configuration that identifies the API resources that are parsed from the
+// user input to either allow or disable propagating them.
+type ResourceConfig struct {
```

Comment:
Having a single Config makes sense since the callers need not change. Thanks for the suggestion. I still renamed `DisabledResourceConfig` to `ResourceConfig` so that we can use for both with a flag.